koptional-reactor-extensions/src/main/kotlin/com/gojuno/koptional/reactor/reactor.kt

```diff
@@ -0,0 +1,36 @@
+package com.gojuno.koptional.reactor
+
+import com.gojuno.koptional.None
+import com.gojuno.koptional.Optional
+import com.gojuno.koptional.Some
+import reactor.core.publisher.Flux
+
+private inline fun <reified R : Any> Flux<*>.ofType(): Flux<R> = ofType(R::class.java)
+
+/**
+ * Filters items emitted by a Flux by only emitting those that are `Some`.
+ * <p>
+ * <img width="640" height="310" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/filter.png" alt="">
+ * <dl>
+ *  <dt><b>Scheduler:</b></dt>
```

Comment:
Let's remove note about scheduling, it's standard for RxJava, but not part of javadoc for operators in Reactor